account-kit/infra/src/middleware/gasManager.ts

```diff
@@ -439,3 +442,29 @@ const requestPaymasterAndData: <C extends ClientWithAlchemyMethods>(
     };
   },
 });
+
+/**
+ * Middleware for managing gas fees with Alchemy's gas manager based on the provided configuration.
+ *
+ * @example
+ *  ```ts
+ * import { alchemyGasManagerMiddleware } from "@account-kit/infra";
+ *
+ * const client = createSmartAccountClient({
+ *      ...
+ *      alchemyErc7677Middleware( {
+ *        policyId,
+ *      })
+ *    );
+ * ```
+ *
+ * @param {AlchemyGasManagerConfig} config the configuration for Alchemy's gas manager
+ * @returns {Pick<ClientMiddlewareConfig, "dummyPaymasterAndData" | "paymasterAndData">} partial client middleware configuration containing `dummyPaymasterAndData` and `paymasterAndData`
+ */
+export function alchemyGasManagerMiddleware(
```

Comment:
let's delete everything else in this file and just keep this in there

Reply:
done. the response structure I can't delete, they still referenced by other code.

Reply:
any code that is still referencing it can be deleted safely as it won't be used anymore